index.js

```diff
@@ -60,6 +60,12 @@ function transform (babel) {
           }
 
           var id = path.node.arguments[0].id || findCandidateNameForExpression(path);
+          
+          // Necessary when some component with forwardRef are using old export model: module.exports
```

Comment:
Passei um tempo quebrando a cabeça aqui, porque o comment está impreciso e o fix parece uma gambiarra. A verdade é que isso de nada tem a ver com usar module.exports: o crash acontece toda vez que o id for do tipo `MemberExpression`.

Um possível fix seria corrigir a função findCandidateNameForExpression para retornar null nesses casos, mas acontece que na verdade de fato faz sentido essa função retornar um id que seja uma MemberExpression: afinal de contas, poderíamos evoluir para usar isso como source para o nome do componente.

Então minha sugestão é ir com um fix aqui, mas outro fix:

```javascript
if (id && id.type !== 'MemberExpression') {
  setDisplayNameUsingClosure(path.get('arguments.0'), id, babel.types);
}
```